api_tests/tests/ether_halight.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @ledger ethereum
+ * @ledger lightning
+ */
+
+import { twoActorTest } from "../src/actor_test";
+import SwapFactory from "../src/actors/swap_factory";
+import { sleep } from "../src/utils";
+
+it(
+    "han-ethereum-ether-halight-lightning-bitcoin-alice-redeems-bob-redeems",
+    twoActorTest(async ({ alice, bob }) => {
+        const [aliceBody, bobBody] = await SwapFactory.newSwap(alice, bob);
+
+        // make sure bob knows about the swap first
```

Comment:
Why? Can the why be explain in the comment please @thomaseizinger

Reply:
Can do yes!